test/stream-muxers.node.js

```diff
@@ -22,14 +22,15 @@ const Switch = require('../src')
 describe('Stream Multiplexing', () => {
   [
     multiplex,
-    spdy
+    spdy // TODO: do we still support this?
```

Comment:
Of course we still support. The whole design of libp2p is to support multiple building blocks.

Reply:
spdy doesn't return any errors when the stream abruptly terminates. We should make the behaviour consistent then. 

What should happen - error on abrupt termination or silent skip?